react/SelectionBar/helpers.js

```diff
@@ -0,0 +1,16 @@
+export function computeMaxAction(maxAction, breakpoints) {
+  if (typeof maxAction === 'object' && maxAction !== null) {
```

Comment:
nit: if maxAction === 'object', it is necessarily !== null no ?

Reply:
This prevent an error on `Object.keys` because in javascript `typeof null` returns `object` ([reason](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/typeof#typeof_null))

Reply:
Wow. I just learnt something...